setup.py

```diff
@@ -3,6 +3,7 @@
 import datetime
 import locale
 import os
+import pip
```

Comment:
this import needs to be after the standard library imports, as `pip` is not a part of the standard library